src/open_inwoner/accounts/managers.py

```diff
@@ -33,6 +33,8 @@ def get_by_rsin(self, rsin):
         return self.get_queryset().get(rsin=rsin)
 
     def eherkenning_create(self, rsin, **kwargs):
+        raise NotImplementedError("old code, please verify before use")
+        # TODO what is this @rsin.com email hostname? @example.org is bad enough but this actually exists
```

Comment:
We don't use eHerkenning at the moment, but we will later this year for companies. Not within scope atm